README.md

```diff
@@ -93,6 +99,9 @@ table.rename(columns={'number': 'decimal'})
 # requires environment variable GOOGLE_APPLICATION_CREDENTIALS
 # or parameter credentials='path-to-credentials.json'
 table.to_bq(table_ref, mode='append')
+
+# with STRCT/RECORD in data schema, use file='json' option for uploading data
+table.to_bq(table_ref, mode='append', file='json')
```

Comment:
it would be more convenient, if we would just switch between .csv- and .json-upload based on the schema implicitly without asking for any parameter.

Reply:
parameter removed.